app/controllers/pages/type_of_answer_controller.rb

```diff
@@ -93,6 +93,6 @@ def set_answer_types
   end
 
   def answer_types
-    Page::ANSWER_TYPES_INCLUDING_FILE
+    Page::ANSWER_TYPES
```

Comment:
Do you reckon it's worth undoing commit [19b5a21faf350ab3208683aaee52bfcd4b81d499](https://github.com/alphagov/forms-admin/commit/19b5a21faf350ab3208683aaee52bfcd4b81d499) and reference the constant from the various places rather than having this method?

I guess the advantage of keeping this method is its easier if we need to have new question types behind a feature flag?